### PR TITLE
fix read the docs build with automatic versioning

### PR DESCRIPTION
**Pull request by JessicaS11**
An alternative to #181 (as suggested there), this adds the setuptools method to the `readthedocs.yml` and successfully builds on read the docs.

### CODE REVIEW

**Comment by lsetiawan** [comment]
@JessicaS11 I guess if there's already a readthedocs automated test, then we can just remove the github workflow one!

**Comment by lsetiawan** [comment]
Thanks for figuring all this out! Sorry I wasn't much help here haha Not super familiar with readthedocs and setuptools_scm! :stuck_out_tongue:

**Comment by JessicaS11** [comment]
> @JessicaS11 I guess if there's already a readthedocs automated test, then we can just remove the github workflow one!

Facepalm. Good point. Okay... here come the edits.

**Comment by weiji14** [comment]
> @JessicaS11 I guess if there's already a readthedocs automated test, then we can just remove the github workflow one!

Was just about to say this too!

> Thanks for figuring all this out! Sorry I wasn't much help here haha Not super familiar with readthedocs and setuptools_scm! stuck_out_tongue

Don't worry Don, I don't think many of us are good with it too, we just keep trying stuff until it works pretty much!

File: doc/source/conf.py [hunk 1]
```diff
@@ -21,6 +22,13 @@
 import recommonmark
 
 
+# -- Versioning with setuptools_scm ------------------------------------------
+from pkg_resources import get_distribution
+release = get_distribution('icepyx').version
+# for example take major/minor
+version = '.'.join(release.split('.')[:2])
+
+
```
Comment (JessicaS11) [inline doc/source/conf.py:31]: So do we need this still? Or can we get rid of it?

Comment (weiji14) [reply doc/source/conf.py:31]: Good question, I tried deleting these lines locally and running `cd icepyx/doc && make html` works. Could you try and remove them and see if the readthedocs build passes still?

**Comment by weiji14** [comment]
Probably best to revert the dev .ipynb notebooks removal too to keep this PR clean:

```
git revert 7a56f48dff896d6a997f4a2290b1b50e67a44399
```

**Review by weiji14 — APPROVED** [review]
Amazing what 2 lines of code can do! Great job team!

**Pull request merged**